LAUNCH.md

```diff
@@ -0,0 +1,10 @@
+# Quick Instructions on Launching the CSSS Website #
+
+The CSSS website can be launched through the following steps:
+
+- SSH into the droplet
+- Enter credentials for root access
+- Enter the folder /opt/csssenv
```

Comment:
before this step, you need to enter the command `source /opt/csssenv/bin/activate`